operator/internal/manifests/internal/gateway/gateway-tenants.yaml

```diff
@@ -79,5 +79,19 @@ tenants:
     withAccessToken: true
 {{- end -}}
 {{- end -}}
+{{- else if eq $l.Stack.Tenants.Mode "openshift-network" -}}
+{{- if $tenant := $l.OpenShiftOptions.Authentication -}}
+{{- range $spec := $l.OpenShiftOptions.Authentication }}
+- name: {{ $spec.TenantName }}
+  id: {{ $spec.TenantID }}
+  openshift:
+    serviceAccount: {{ $spec.ServiceAccount }}
+    redirectURL: {{ $spec.RedirectURL }}
+    cookieSecret: {{ $spec.CookieSecret }}
+  opa:
+    url: {{ $l.OpenShiftOptions.Authorization.OPAUrl }}
+    withAccessToken: true
+{{- end -}}
+{{- end -}}
```

Comment:
I suggest to collapse this with the block for `openshift-logging` and make the guarding else if to accept `openshift-logging` or `openshift-network` as tenants mode.

Reply:
Agreed. I was not really comfortable on that part. Could you please help me on this ?

Reply:
Sure you change L68 to something like that:
```go
{{- else if (or (eq $l.Stack.Tenants.Mode "openshift-logging") (eq $l.Stack.Tenants.Mode "openshift-network")) -}}
```